libr/util/sys.c

```diff
@@ -73,6 +73,7 @@ extern char **environ;
 #include <psapi.h>
 #include <process.h>  // to allow getpid under windows msvc compilation
 #include <direct.h>  // to allow getcwd under windows msvc compilation
+#define sig_t int
```

Comment:
I think this is wrong. `signal` does not return a int and it was already fixed in https://github.com/radareorg/radare2/commit/83d3e7bf1872b93af059f1f912d2e5199c8b51dd